example/lib/main.dart

```diff
@@ -23,22 +23,26 @@ class MyApp extends StatelessWidget {
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    eachFrame().take(10000).transform(const ComputeFps()).listen((fps) => print('fps: $fps'));
+    eachFrame()
+        .take(10000)
+        .transform(const ComputeFps())
+        .listen((fps) => print('fps: $fps'));
```

Comment:
Do you really want to print this to the console? Otherwise use drain instead.

Reply:
I don't think we need to print it, it was something from the initial package. I'll remove.